src/gui/cloudproviders/cloudproviderwrapper.cpp

```diff
@@ -221,7 +221,14 @@ GMenuModel* CloudProviderWrapper::getMenuModel() {
     _mainMenu = g_menu_new();
 
     section = g_menu_new();
-    item = addMenuItem(tr("Open website"), "cloudprovider.openwebsite");
+    item = addMenuItem(tr("Open main dialog"), "cloudprovider.openmaindialog");
```

Comment:
"Main dialog" is quite technical and I had no idea what this would do before reading the issue.

Just "Open Nextcloud Desktop" would be better imo.

It should also probably be the last context menu item instead of the first as I don't think it would be used a lot (since it is not actually contextual to a file, same as eg. Help)

Reply:
The same text [appears in the tray icon context menu](https://github.com/user-attachments/assets/42ead78a-223e-4e2b-807a-2aa465cf03a2) (on Windows and some Linux desktops at least), so I guess we should we change that one as well then...

Reply:
I did change the action title as suggested